### DISCUSSION TITLE
compact windows 10 style taskbar icons option?👉👈

### DISCUSSION THREAD

**Discussion by chloechillworld** [valinet/ExplorerPatcher#657] [Ideas]
title

**Comment by valinet** [comment]
???

**Reply by chloechillworld** [reply]
how to make opened programs look like this in the taskbar
![task bar](https://user-images.githubusercontent.com/94586008/149216699-f2d00e6f-e91c-4e68-87ba-0daab1e44dc9.png)

**Reply by valinet** [reply]
Right click taskbar, Properties, it’s one of the options in the Taskbar section. Please read the README as well, it directed you there already.

**Reply by chloechillworld** [reply]
thankyou
